stix_shifter_modules/gcp_chronicle/stix_translation/json/to_stix_map.json

```diff
@@ -0,0 +1,1702 @@
+        "key": "x-ibm-finding.extensions.x-gcp-chronicle-security-result.threat_status",
+        "object": "security_result"
+      },
+      "description": {
+        "key": "x-ibm-finding.extensions.x-gcp-chronicle-security-result.description",
+        "object": "security_result"
+      },
+      "action": {
+        "key": "x-ibm-finding.extensions.x-gcp-chronicle-security-result.actions_taken",
+        "object": "security_result"
+      }
+    },
+    "metadata": {
+      "eventType": {
+        "key": "x-oca-event.action",
+        "object": "event"
+      },
+      "productLogId": {
+        "key": "x-oca-event.code",
+        "object": "event"
+      },
+      "eventTimestamp": [
+        {
+          "key": "x-oca-event.created",
+          "object": "event"
+        },
+        {
+          "key": "first_observed",
+          "cybox": false
+        },
+        {
+          "key": "last_observed",
+          "cybox": false
+        }
+      ],
+      "description": {
+        "key": "x-oca-event.extensions.x-gcp-chronicle-event.description",
+        "object": "event"
+      },
+      "productName": {
+        "key": "x-oca-event.agent",
+        "object": "event"
+      },
+      "vendorName": {
+        "key": "x-oca-event.provider",
+        "object": "event"
+      },
+      "productEventType": {
+        "key": "x-oca-event.outcome",
+        "object": "event"
+      }
+    },
+    "about": {
+      "file": {
+        "fullPath": [
+          {
+            "key": "file.name",
+            "object": "about_file",
+            "transformer": "ToFileName"
+          },
+          {
+            "key": "directory.path",
+            "object": "about_directory",
+            "transformer": "ToDirectoryPath"
+          },
+          {
+            "key": "file.parent_directory_ref",
+            "object": "about_file",
+            "references": "about_directory"
+          },
+          {
+            "key": "email-message.extensions.x-gcp-chronicle-email-message.file_ref",
+            "object": "em",
+            "references": "about_file"
+          }
+        ],
+        "size": {
+          "key": "file.size",
+          "object": "about_file",
+          "transformer": "ToInteger"
+        },
+        "md5": {
+          "key": "file.hashes.MD5",
+          "object": "about_file"
+        },
+        "sha1": {
+          "key": "file.hashes.SHA-1",
+          "object": "about_file"
+        },
+        "sha256": {
+          "key": "file.hashes.SHA-256",
+          "object": "about_file"
+        },
+        "mimeType": {
+          "key": "file.extensions.x-gcp-chronicle-file.mime_type",
+          "object": "about_file"
+        },
+        "lastModificationTime": {
+          "key": "file.modified",
+          "object": "about_file"
+        },
+        "fileType": {
+          "key": "file.extensions.x-gcp-chronicle-file.file_type",
+          "object": "about_file"
+        }
+      }
+    },
+    "additional": {
```

Comment:
are these fields only specific to aws events? is chronicle always receives logs from aws? these are very odd extension properties.

Reply:
These fields are specific to AWS.  The “additional” key which belongs to UDM schema, doesn’t have any standard fields . It may vary between log sources.  Since observed this additional information in AWS events which had extra information like detector id and image id, we have added this information in to_stix mapping alone. If additional information needs to be added in future other log sources also, it  can add here.

Reply:
This is fine for now.  we may have think about handling "additional" properties of other datasets from a different product.